src/sentry/rules/actions/sentry_apps/notify_event.py

```diff
@@ -68,47 +72,36 @@ def _get_sentry_app_installation_uuid(self) -> Any:
             raise ValidationError("Missing attribute 'sentryAppInstallationUuid'")
         return sentry_app_installation_uuid
 
-    def _get_alert_rule_component(self, sentry_app_id: int, sentry_app_name: str) -> Any:
-        try:
-            alert_rule_component = SentryAppComponent.objects.get(
-                sentry_app_id=sentry_app_id, type="alert-rule-action"
-            )
-        except SentryAppComponent.DoesNotExist:
-            raise ValidationError(
-                f"Alert Rule Actions are not enabled for the {sentry_app_name} integration."
-            )
-        return alert_rule_component
+    def _get_alert_rule_component(
+        self, sentry_app_id: int, sentry_app_name: str
+    ) -> RpcSentryAppComponent:
+        components = app_service.find_app_components(app_id=sentry_app_id)
```

Comment:
Not necessary to do now...but I think this RPC is avoidable. iirc, fetching the app installation from the service already fetches the sentry_app and sentry app components, so we can just filter them here.

Reply:
I do agree and there are several places that can be done.  I wanted to keep the changeset "safe" first and foremost.